src/GovUk.Education.ExploreEducationStatistics.Common.Tests/Converters/TimeIdentifierCategoryJsonConverterTests.cs

```diff
@@ -0,0 +1,138 @@
+using GovUk.Education.ExploreEducationStatistics.Common.Converters;
+using GovUk.Education.ExploreEducationStatistics.Common.Model;
+using Newtonsoft.Json;
+using Xunit;
+
+namespace GovUk.Education.ExploreEducationStatistics.Common.Tests.Converters
+{
+    public class TimeIdentifierCategoryJsonConverterTests
+    {
+        private class SampleClass
+        {
+            public string StringFieldBefore { get; set; }
+
+            [JsonConverter(typeof(TimeIdentifierCategoryJsonConverter))]
+            public TimeIdentifierCategory? SampleField { get; set; }
+
+            public string StringFieldAfter { get; set; }
+
+            protected bool Equals(SampleClass other)
+            {
+                return string.Equals(StringFieldBefore, other.StringFieldBefore) && SampleField == other.SampleField &&
+                       string.Equals(StringFieldAfter, other.StringFieldAfter);
+            }
+
+            public override bool Equals(object obj)
```

Comment:
Do we need these equals methods?

Reply:
The make the equality tests a bit easier - no need to check each field in each test is all